src/python/WMCore/MicroService/MSUnmerged/MSUnmergedRSE.py

```diff
@@ -0,0 +1,49 @@
+"""
+File       : MSUnmergedRSE.py
+Description: Provides a document Template for the MSUnmerged MicroServices
+"""
+
+
+class MSUnmergedRSE(dict):
+    """
+    A minimal RSE information representation to serve the needs
+    of the MSUnmerged Micro Service.
+    """
+    def __init__(self, rseName, **kwargs):
+        super(MSUnmergedRSE, self).__init__(**kwargs)
+
+        """
+        Document Schema:
+        {
+        "name": "RSEName",
+        "delInterface": "",
+        "isClean": False,
+        "counters": {"totalNumFiles" : 0,
+                      "numFilesDeleted" : 0},
+        "files": {"allUnmerged": [],
+                  "toDelete": [],
+                  "protected": [],
+                  "deletedSuccess": [],
+                  "deletedFail": []},
+        "dirs":  {"allUnmerged": [],
+                  "nonEmpty": [],
+                  "empty": []}
+        }
+        """
+        myDoc = {
+            "name": rseName,
+            "delInterface": "",
+            "isClean": False,
+            "counters": {"totalNumFiles" : 0,
```

Comment:
I like this schema!
Just one question on the counters/totalNumFiles. Is it meant to be total number of files eligible for deletion? Or total number of files in the unmerged area (meaning, not necessarily eligible for deletion)?

Reply:
Good question. This is a sign that the name chosen was not expressive enough, so it does not contain the needed amount of information in it. I should be total files eligible for deletion. It is used in the `_checkClean` method it is also returned to the report document (the string there tells it) ... but ye the name is maybe not the best. - I was following the general login in the report document and didn't think that would lead to the current confusion.

Reply:
As we discussed offline, this is meant to be the total amount of deletable files (files that pass the WMStats filter). I gave you a suggestion this afternoon, but you could add a comment above and it should be enough as well.

Reply:
note made